packages/api-client/src/components/ViewLayout/ViewLayoutCollapse.vue

```diff
@@ -17,11 +17,11 @@ withDefaults(
   <Disclosure
     v-slot="{ open }"
     as="div"
-    class="focus-within:bg-b-2 ui-not-open:hover:bg-b-2 focus-within:text-c-1 text-c-2 rounded request-item ui-not-open:bg-transparent ui-open:pb-1 ui-open:[&:not(:last-child)]:mb-1.5 ui-not-open:mb-0 ui-not-open:pb-0"
+    class="focus-within:bg-b-2 ui-not-open:hover:bg-b-2 focus-within:text-c-1 text-c-2 rounded request-item ui-not-open:bg-transparent ui-open:[&:not(:last-child)]:mb-2.5 ui-not-open:mb-[0.5px]"
     :defaultOpen="defaultOpen">
     <div class="flex items-center">
       <DisclosureButton
-        class="group hover:text-c-1 flex flex-1 items-center gap-2.5 overflow-hidden py-1.5 text-sm font-medium px-1.5 outline-none">
+        class="group hover:text-c-1 flex flex-1 items-center gap-2.5 overflow-hidden py-1.5 text-sm font-medium px-2.5 outline-none">
```

Comment:
nitpick: could use diff padding here to maintain caret alignment:

```suggestion
        class="group hover:text-c-1 flex flex-1 items-center gap-2.5 overflow-hidden py-1.5 text-sm font-medium px-1 md:px-1.5 xl:px-2 outline-none">
```

Reply:
feel free to change it :)